tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -1826,6 +1940,70 @@ public function test_get_items_pagination_headers() {
 		$this->assertStringContainsString( '<' . $next_link . '>; rel="next"', $headers['Link'] );
 	}
 
+	/**
+	 * Data provider intended to provide HTTP method names for testing GET and HEAD requests.
+	 *
+	 * @return array
+	 */
+	public function data_readable_http_methods() {
+		return array(
+			'GET request'  => array( 'GET' ),
+			'HEAD request' => array( 'HEAD' ),
+		);
+	}
+
+	/**
+	 * @dataProvider data_readable_http_methods
+	 * @ticket 56481
+	 *
+	 * @param string $method HTTP method to use.
+	 */
+	public function test_get_items_returns_only_fetches_ids_for_head_requests( $method ) {
```

Comment:
```suggestion
	public function test_get_items_returns_only_ids_for_head_requests( $method ) {
```
Suggestion per above.

Reply:
https://github.com/WordPress/wordpress-develop/pull/7970#discussion_r1876900479